pages/order-status/index.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <div id="order-status-container">
+    <SfBreadcrumbs class="breadcrumbs" :breadcrumbs="breadcrumbs" />
+    <div class="order-status-title">
+      <div class="header-text-weight">{{ $t("View Order Status") }}</div>
+    </div>
+    <hr class="order-status-hr" />
+    <div v-show="!isOpenOrderStatus" class="order-status">
+      <div class="status-order">
+        {{ $t("Checking the status of your order is fast and simple!") }}
+      </div>
+      <div class="track-order">
+        {{ $t("Simply enter your order number and billing email to track your order.") }}
```

Comment:
Have you updated lang/en.js file? I don't see this file in PR

Reply:
Resolved